src/test/java/Salma_BaseTest.java

```diff
@@ -0,0 +1,139 @@
+
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.time.Duration;
+
+
+public class Salma_BaseTest {
+    WebDriver driver;
+    WebDriverWait wait;
+    String url;
+    Actions actions;
+
+
+    @BeforeSuite
+    public static void chromeConfigs() {
+        // This is for Windows users
+        if (System.getProperty("os.name").toLowerCase().contains("win")) {
+            System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        }
+    }
+    @BeforeMethod
+    @Parameters({"baseURL"})
+    public void launchBrowser(String baseURL) throws MalformedURLException {
+        //driver = new ChromeDriver();
+      //System.setProperty("webdriver.gecko.driver", "geckodriver");
+       //driver= new FirefoxDriver();
+        //driver = new SafariDriver();
+        driver = pickBrowser(System.getProperty("browser"));
+        //driver = pickBrowser("firefox");
+        //driver = pickBrowser("safari");
+        actions = new Actions(driver);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        wait= new WebDriverWait(driver, Duration.ofSeconds(5));
+        url = baseURL;
+        driver.get(url);
+
+    }
+
+    private WebDriver pickBrowser(String browser) throws MalformedURLException {
```

Comment:
description over method 
//we got this by running "java -jar selenium-server-4.6.0.jar standalone" in terminal